src/galaxy/StarSystem.cpp

```diff
@@ -109,35 +110,35 @@ const Color StarSystem::starRealColors[] = {
 	{ 255, 153, 153 }, // M WF
 	{ 204, 204, 255 }, // B WF
 	{ 255, 204, 255 }, // O WF
-	{ 22, 0, 24 }, // small Black hole
-	{ 16, 0, 20 }, // med BH
-	{ 10, 0, 16 } // massive BH
+	{ 22, 0, 24 },	   // small Black hole
+	{ 16, 0, 20 },	   // med BH
+	{ 10, 0, 16 }	   // massive BH
 };
 
 const double StarSystem::starLuminosities[] = {
 	0,
-	0.0003, // brown dwarf
-	0.1, // white dwarf
-	0.08, // M0
-	0.38, // K0
-	1.2, // G0
-	5.1, // F0
-	24.0, // A0
-	100.0, // B0
-	200.0, // O5
-	1000.0, // M0 Giant
-	2000.0, // K0 Giant
-	4000.0, // G0 Giant
-	6000.0, // F0 Giant
-	8000.0, // A0 Giant
-	9000.0, // B0 Giant
-	12000.0, // O5 Giant
-	12000.0, // M0 Super Giant
-	14000.0, // K0 Super Giant
-	18000.0, // G0 Super Giant
-	24000.0, // F0 Super Giant
-	30000.0, // A0 Super Giant
-	50000.0, // B0 Super Giant
+	0.0003,	  // brown dwarf
+	0.1,	  // white dwarf
+	0.08,	  // M0
+	0.38,	  // K0
+	1.2,	  // G0
+	5.1,	  // F0
+	24.0,	  // A0
+	100.0,	  // B0
+	200.0,	  // O5
+	1000.0,	  // M0 Giant
+	2000.0,	  // K0 Giant
+	4000.0,	  // G0 Giant
+	6000.0,	  // F0 Giant
+	8000.0,	  // A0 Giant
+	9000.0,	  // B0 Giant
+	12000.0,  // O5 Giant
+	12000.0,  // M0 Super Giant
```

Comment:
On Github this looks unaligned with the above data. @impaktor didn't you use to have a trick to make the GH code view behave sensibly ?

Reply:
Github's tabstops are weird, they don't respect our editorconfig file. You can set your tab width to 4 IIRC and that fixes things.

It looks perfectly fine in a 4-TAB text editor, so I'm going to call it an acceptable weirdness.